### DISCUSSION TITLE
Is there a way to esclude by column indexes?

### DISCUSSION THREAD

**Discussion by aborruso** [duckdb/duckdb#13632] [Q&A]
Hi,
I can run `SELECT #1, #3 FROM tbl`, but not `SELECT * EXCLUDE(#3) FROM tbl`.

Is there another small and simple way to exclude a column by its index?

Thank you

**Comment by Tishj** [comment]
I dont think there is but it sounds like a solid feature request to support positional parameters here

**Reply by aborruso** [reply]
Hi @Tishj thank you.

My feature request: https://github.com/duckdb/duckdb/discussions/13680
